pkg/apis/camel/v1/integration_types.go

```diff
@@ -155,7 +155,13 @@ const (
 	IntegrationPhaseRunning IntegrationPhase = "Running"
 	// IntegrationPhaseError --.
 	IntegrationPhaseError IntegrationPhase = "Error"
+	// IntegrationPhaseImportMissing used when the application from which the Integration is imported has been deleted.
+	IntegrationPhaseImportMissing IntegrationPhase = "Application Missing"
+	// IntegrationPhaseCannotMonitor used when the application from which the Integration has not enough information to monitor its pods.
+	IntegrationPhaseCannotMonitor IntegrationPhase = "Cannot Monitor Pods"
```

Comment:
I rather use conditions than phases

Reply:
For both "Application Missing" and "Cannot Monitor Pods"? Which should be the phase in this case? I think neither `Ready` or `Error`.

Reply:
I think that if the deployment is mis configured, then the integration should be in an error status and a condition should report the reason.